WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -412,7 +412,7 @@ public async Task RegisterOutputAsync(OutputRegistrationRequest request)
 				var outputValue = bob.CalculateOutputAmount(round.FeeRate);
 
 				var vsizeCredentialRequests = request.VsizeCredentialRequests;
-				if (-vsizeCredentialRequests.Delta != bob.OutputVsize)
+				if (-vsizeCredentialRequests.Delta < bob.OutputVsize)
```

Comment:
what motivated this change? the dependency graph resolver should not produce any requests where the vsize doesn't match exactly, so this suggests a bug in that code if it was necessary to work around something

Reply:
There are at least 15 tests that don't pass without this change. I didn't know the dependency graph produces exact values for vsize credentials too.

Reply:
It can be  a bug in this PR. Checking.....

Reply:
the same algorithm is used for both credential types, the only difference is the order, amount credentials are resolved before vsize credentials which can affect things because of how reissuance nodes are created.

if it fails to exactly satisfy the boundary condition specified by the input and output values (of both types), that's a bug we should investigate even if we decide to relax so that behaves like you modified it here (argument for: simpler, more efficient. against: adds some weird edge cases to re-planning/error recovery)